nexus/db-queries/src/db/datastore/rack.rs

```diff
@@ -1334,22 +1335,22 @@ mod test {
 
         let external_dns_ip = IpAddr::V4(Ipv4Addr::new(1, 2, 3, 4));
         let external_dns_pip = DNS_OPTE_IPV4_SUBNET
-            .nth(NUM_INITIAL_RESERVED_IP_ADDRESSES as u32 + 1)
```

Comment:
Very happy to get rid of the use of `as`.

Reply:
Good news: I added  more back in

Reply:
lol